src/App.jsx

```diff
@@ -1,14 +1,52 @@
 import './App.css';
+import ChatLog from './components/ChatLog';
+import chatData from './data/messages.json';
+import { useState } from 'react';
+
+// Helper function to count total likes
+const calculateTotalLikes = (chatData) => {
+  let total = 0;
+  for (const like of chatData) {
+    total += like.liked;
+  }
+  return total;
+};
```

Comment:
Could also use `reduce`
```suggestion
const calculateTotalLikes = () => {
  return likeData.reduce((totalLikes, message) => (totalLikes + message.liked), 0);
};
```

Reply:
Did you mean to write `calculateTotalLikes` outside of the App component?

I'd expect it to be written inside the app component, after line 16.

Then you can directly access the state `likeData` which holds all your chat messages and you wouldn't need to pass in an arg to `calculateTotalLikes`